.github/workflows/main.yml

```diff
@@ -24,65 +24,79 @@ jobs:
           compiler_version: "5"
           python: 2.7
```

Comment:
This file will not be checked in. I'm using this to reduce the number of builds per pull requests.